web/help/rlt-notes-unit1.md

```diff
@@ -0,0 +1,14 @@
+If you do not leave a space before and after the & the popup does not show
```

Comment:
I'm not sure what you mean by "the popup".  In general, I don't think spaces around operators like `&` should make a difference, so I'd love to see what example you're thinking of here.

Reply:
Ignore it. I thought I noticed that when there were no spaces around the &,, even though the program ran, the popup explanation that appears above the & when the mouse hovers over was not showing. I was mistaken. It shows.